pkg/domain/infra/abi/play.go

```diff
@@ -560,6 +562,8 @@ func (ic *ContainerEngine) playKubePod(ctx context.Context, podName string, podY
 		infraImage := util.DefaultContainerConfig().Engine.InfraImage
 		infraOptions := entities.NewInfraContainerCreateOptions()
 		infraOptions.Hostname = podSpec.PodSpecGen.PodBasicConfig.Hostname
+		infraOptions.ReadOnly = true
+		infraOptions.ReadWriteTmpFS = false
```

Comment:
To be sure: the service-container sets it `true`, the infra to `false`.  Is that intentional? Both execute the same work.

Reply:
The both should set it False now, but yes the same logic applies.

Reply:
The service container sets it to `true` though (https://github.com/containers/podman/pull/16545/files#diff-d3fb054de60fb7550ae0554fc1c99ac63b8434f960b49b5b7f0352f1d0fcb8f0R72).

Reply:
Shoot.